debian/postinst

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+cat << HEREDOC > /etc/ood/profile
+#!/bin/bash
+
+export GEM_PATH="/opt/ood/gems:$GEM_PATH"
+HEREDOC
+
+cat << HEREDOC > /etc/sudoers.d/ood
```

Comment:
Is this sudo file not packages, but added on post install? Wouldn't it be better to package so it's clear that file is owned by the package?  With RPMs you can mark files as "configs" which will not be replaced if modified after install.

Reply:
Yea I can move that to debian/control.

Reply:
I would recommend any files like the `profile` one too get added into package.  It tends to be better for sysadmins to have files in the package than having to look at post install scripts to figure out where files come from.  If I can run like "apt-file find ..." on something , that's a lot better than whatever is done to dump postinst and see where files come from.

Reply:
Moved both of these to `lib/tasks/files/` with a `.debian` extension. It's not the best place, but we can figure out something more suitable after we merge into master.